doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -448,107 +438,134 @@ Libraries for networking
 
     * Updated:
 
-      * During the connection process, shadow data is sent to the application even if no "config" section is present.
+      * During the connection process, shadow data is sent to the application even if no ``"config"`` section is present.
       * The application can now send shadow updates earlier in the connection process.
       * nRF Cloud error message responses to location service MQTT requests are now handled.
 
-    * Fixed:
-
-      * Validation of bootloader FOTA updates.
+    * Fixed the validation of bootloader FOTA updates.
 
   * :ref:`lib_aws_iot` library:
 
-    * Renamed ``aws_iot_topic_type`` to ``aws_iot_shadow_topic_type`` and removed ``AWS_IOT_SHADOW_TOPIC_UNKNOWN``.
+    * Updated:
+
+      * Renamed ``aws_iot_topic_type`` to ``aws_iot_shadow_topic_type`` and removed ``AWS_IOT_SHADOW_TOPIC_UNKNOWN``.
 
   * :ref:`lib_lwm2m_client_utils` library:
 
-    * Updated the library to store credentials and server settings permanently on bootstrap.
     * Added support for using location assistance when using Coiote LwM2M server.
+    * Updated the library to store credentials and server settings permanently on bootstrap.
+
+
+
+Libraries for NFC
+-----------------
+
+* Updated:
+
+  * :ref:`nfc_ndef_parser_readme`:
+
+    * Updated:
+
+      * :c:func:`nfc_ndef_msg_parse` - fixed declaration, added an assertion to avoid a potential usage fault and added a note in the function documentation.
```

Comment:
```suggestion
      * :c:func:`nfc_ndef_msg_parse` with a fix to declaration, a new assertion to avoid a potential usage fault, and a note in the function documentation.
```

Reply:
fixed